java/FileCache/README.md

```diff
@@ -0,0 +1,19 @@
+# Flink File Cache usage example
+
+* Flink version: 1.20
+* Flink API: DataStream API
+* Language: Java (11)
+* Flink connectors: DataGeneratorSource, DiscardingSink
+
+
+This example demonstrate how to use Flink's distributed cache to copy files over from JobManager and distribute them to the TaskManager worker nodes.
```

Comment:
We need to explain clearly when this is useful, because this example doesn't show using the certificate. It just copies it. 
In particular, we are showing downloading a cacert and not just random file. We need to explain when is useful